src/Interface/Application/SCIRunMainWindowQtOverrides.cc

```diff
@@ -108,6 +108,7 @@ void SCIRunMainWindow::closeEvent(QCloseEvent* event)
   {
     writeSettings();
     event->accept();
+    quit();
```

Comment:
Is this still needed with the executionManager.stop call? It's risky since it's buried in a conditional, and might not be needed on other platforms (we don't want to call quit() twice)

Reply:
On second thought, this might not be necessary.

Reply:
Test it out without the extra quit and see if it works

Reply:
I don't have any problems without it. I thought I was getting a bug without that call. I reverted some code before committing so I think it came from there.